src/main/java/com/deloitte/elrr/datasync/controller/ImportsController.java

```diff
@@ -42,6 +42,13 @@ public class ImportsController {
   @GetMapping("/getImports")
   public ImportDTO getImports(
     @RequestParam(value = "name", required = true) final String importsName) {
+
+    String regex = "^[a-zA-Z0-9-]+$";
```

Comment:
This regex will check for alpha-numerics and hyphon

Reply:
Yes, we should have tests for this change.  I can add missing tests for the entire code base as part of that story, if thats fine.